### PR TITLE
Added a --visualize option to generate build output stats

### PR DESCRIPTION
**Pull request by AlexHayton**
Following up from https://github.com/developit/microbundle/issues/526

Updated readme
Added the new visualize flag
Added a test with a postbuild script that copies the stats.html
Allow tests to have a postbuild script! :)

### CODE REVIEW

**Comment by AlexHayton** [comment]
Hmm, I really can't get my snapshots to match travis. Running on OSX here. Does anyone have any idea what's going on?

File: test/fixtures/visualize/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "visualize",
+  "scripts": {
+    "build": "microbundle --visualize",
+    "postbuild": "mv stats.html dist/"
```
Comment (marvinhagemeister) [inline test/fixtures/visualize/package.json]: I wonder if we can remove the need for a `postbuild` task. The docs of `rollup-plugin-visualize` don't mention it, but you can pass a directory with the `filename` setting.

https://github.com/btd/rollup-plugin-visualizer/blob/41aeee4741ab424c8e4e1ab48930ca5cbd2d8fb1/plugin/index.js#L129

Comment (AlexHayton) [reply test/fixtures/visualize/package.json]: Sounds like a good plan.

@marvinhagemeister Do you have any idea about the other (unrelated) snapshots that are failing?
Last time I checked master is also failing too... so not sure what to do next.

**Comment by developit** [comment]
@AlexHayton sorry for the slow reply - I also have the issue with snapshots being out of sync on Mac OS. Unfortunately, I haven't yet found a fix for this :/

**Comment by AlexHayton** [comment]
Sorry I've been away from this for a while. Life got in the way. I'll try and resubmit soon. I worked out why the snapshots were failing, it is an issue with the randomly generated CSS class names in the visualiser output. So I am hoping it's possible to pass a specific seed or just take those out of the snapshots...

**Comment by AlexHayton** [comment]
> @AlexHayton sorry for the slow reply - I also have the issue with snapshots being out of sync on Mac OS. Unfortunately, I haven't yet found a fix for this :/

@developit I think found a way around this, I will submit it seperately...

**Comment by developit** [comment]
Just an update: this should all work without the Docker shennanigans now that we're on GH actions. Sorry!

**Comment by AlexHayton** [comment]
Wow, zombie thread back from the dead 😄 
I'll give it another go, thank you!

**Comment by AlexHayton** [comment]
Actually reading my past self's comments, the specific blocker here was the CSS class names...

**Comment by developit** [comment]
No pressure btw! I like the idea of this feature though, and I'd be happy to rebase - I don't have the snapshots issues when working in the repo anymore (perhaps they're still there...)

**Comment by rschristian** [comment]
Ended up implementing in #549 rather than editing this PR directly. It's finally merged!

**Pull request closed without merging**